src/definitions/modules/slider.less

```diff
@@ -110,11 +110,15 @@
   right: 0;
 }
 
-.ui.reversed.slider .inner .thumb {
+.ui.reversed.slider:not(.vertical) .inner .thumb {
   left: auto;
   right: 0;
 }
 
+.ui.reversed.vertical.slider .inner .thumb {
+    left: 0.03em;
```

Comment:
Please use a variable like `@verticalSliderThumbOffset` and define it in slider.variables instead of hardcoding it here